asm/macros/scrcmd.inc

```diff
@@ -403,9 +403,9 @@
     .byte \arg0
     .endm
 
-    .macro ScrCmd_049 arg0
+    .macro PlayFanfare sdatSeq
```

Comment:
I like that better than `sdatSeq`. I got the name from the one used in `BtlCmd_PlaySound()`, `NNS_SndArcPlayerStartSeq`, and the general `SEQ_XYZ` naming scheme in `constants/sdat.h` (though I kinda doubt we'll stick to that). 

I'm also now noticing the sdat enum itself is called `SDATID`. I like `sdatID` since it's consistent with other `xyzID` names for that sort of thing in the repo. What do you think about that?

Reply:
I also like `sdatID`; if you want to standardize it in the code for the other two functions, that would also be much-appreciated!